src/components/project-card.tsx

```
@@ -0,0 +1,80 @@
+import Link from "next/link"
+import { Card, CardContent, CardFooter, CardHeader, CardTitle } from "@/components/ui/card"
+import { Button } from "@/components/ui/button"
+import { Progress } from "@/components/ui/progress"
+import { BookOpen, Leaf, Cpu, Palette, Heart, Users, LayoutGrid, Building2 } from "lucide-react"
+
+const categoryIcons = {
+  All: LayoutGrid,
```

ESLint

[error] Inconsistently quoted property 'All' found.

(style/quote-props)